.github/workflows/.deploy.yml

```diff
@@ -84,14 +84,21 @@ jobs:
             echo "tag=${{ inputs.tag }}" >> $GITHUB_OUTPUT
           fi
 
-          if [ -z "${{ inputs.url }}" ]; then
-            echo "Creating URL"
-            echo "url=fom-$(( ${{ inputs.target }} % 50 )).apps.silver.devops.gov.bc.ca"
-            echo "url=fom-$(( ${{ inputs.target }} % 50 )).apps.silver.devops.gov.bc.ca" >> $GITHUB_OUTPUT
-          else
+          if [ ! -z "${{ inputs.url }}" ]; then
             echo "Using URL parameter"
             echo "url=${{ inputs.url }}"
             echo "url=${{ inputs.url }}" >> $GITHUB_OUTPUT
+          elif [ -z "${{ inputs.url }}" ] && [ '${{ github.event_name }}' == 'pull_request' ]; then
+            echo "Creating URL from inputs.target (= github.event.number) mod 50"
+            echo "url=fom-$(( ${{ inputs.target }} % 50 )).apps.silver.devops.gov.bc.ca"
+            echo "url=fom-$(( ${{ inputs.target }} % 50 )).apps.silver.devops.gov.bc.ca" >> $GITHUB_OUTPUT
+          elif [ -z "${{ inputs.url }}" ] && [ '${{ github.event_name }}' != 'pull_request' ]; then
```

Comment:
Same as above if the single quote/double quote matter?

Also, not sure if we need the first expression `-z "${{ inputs.url }}"` because the first condition is `! -z "${{ inputs.url }}" ` so after `then` the rest of the conditions probably imply it is  `-z "${{ inputs.url }}"` (again, not very sure, not familiar with shell...)

Reply:
Good catch!  Redundant code removed.  :)